package.json

```
@@ -2,10 +2,21 @@
   "name": "bitmovin-player-ui",
   "version": "4.0.0-alpha.1",
   "description": "Bitmovin Player UI Framework",
-  "main": "./dist/js/framework/main.js",
+  "main": "./dist/js/bitmovinplayer-ui.js",
   "types": "./dist/js/framework/main.d.ts",
   "scripts": {
     "test": "npx jest",
+    "start": "webpack-dev-server --mode development --open",
+    "build": "npm run clean; npx webpack --mode development",
+    "build:prod": "npm run clean; npm run lint && npx webpack --mode production && npm run copy-json && npm run docs",
+    "lint": "npm run lint-ts && npm run lint-sass",
+    "format": "npx prettier --write *",
+    "docs": "npx typedoc",
+    "clean": "rm -r dist",
```

Nit: just to be sure it always works

```suggestion
    "clean": "rm -rf dist",
```

---

Shouldn't be necessary, but sure, why not 😄 done in a54c3e19